src/app/person.model.ts

```diff
@@ -0,0 +1,44 @@
+export interface Person {
+    /**
+     * UUID. Set on creation.
+     */
+    id?: string;
+
+    /**
+     * Required for creation.
+     */
+    captcha_code?: string;
+
+    /**
+     * These 3 expected on first update.
+     */
+    email_address?: string;
+    first_name?: string;
+    last_name?: string;
+
+    /**
+     * These 3 expected if donor's claiming Gift Aid.
+     */
+    home_address_line_1?: string;
+    home_postcode?: string;
+    home_country_code?: string;
+
+    /**
+     * Set on update if the donor opts to make a resuable account.
+     */
+    raw_password?: string;
```

Comment:
@NoelLH Isn't the password hashed when stored in the ID DB? If so, I would advise renaming this variable to `hashed_password`.

Reply:
@alihejazi This wouldn't be correct because the password sent from the client is only ever the raw password. Additionally, the API will never returned the hashed password: it is used only for server-side validation of login attempts.